MaxText/configs/mistral-7b.yml

```diff
@@ -0,0 +1,19 @@
+base_config: "base.yml"
+
+# Args coming from the NVIDIA spreadsheet http://shortn/_W9CzVbtQde and
```

Comment:
I see mixtral-8x22b.yml added in both configs/mixtral_8x22b.yml and configs/models/mixtral_8x22b.yml. I think all of the these under /configs instead of /configs/models should be removed?

Reply:
Yes! I believe I was working with an old branch before rebasing.